### PR TITLE
#271: fixed MsetCmdMeta to support COMMAND getkeys

### PR DESCRIPTION
**Pull request by SyedMa3**
Fixes: Issue #271 

Added `KeySpec.LastKey = -1` to support `COMMAND getkeys MSET`

### CODE REVIEW

File: core/commands.go [hunk 1]
```diff
@@ -62,7 +62,7 @@ var (
 		Returns encoded OK RESP once all entries are added`,
 		Eval:     evalMSET,
 		Arity:    -3,
-		KeySpecs: KeySpecs{BeginIndex: 1, Step: 2},
+		KeySpecs: KeySpecs{BeginIndex: 1, Step: 2, LastKey: -1},
```
Comment (lucifercr07) [inline core/commands.go:65]: Can we please add a test case for this, or create an issue so that we can keep track of this.

Comment (SyedMa3) [reply core/commands.go:65]: Done

**Comment by Yonggiee** [comment]
I wrote the `COMMAND GETKEYS`, should be correct

**Review by JyotinderSingh — APPROVED** [review]
Thanks for the fixes and test improvements @SyedMa3. Thanks for the reviews @lucifercr07 and @Yonggiee.

Approved, Merging.

**Pull request merged**